cmd/root.go

```diff
@@ -76,6 +76,7 @@ Learn more about Knative at: https://knative.dev`, cfg.Name),
 				NewDeployCmd(newClient),
 				NewDeleteCmd(newClient),
 				NewListCmd(newClient),
+				NewSubscribeCmd(),
```

Comment:
For testing support, this should pass through the `newClient` client factory; but I don't think that should hold up the PR.

Reply:
I will add additional PR. thanks for the review on this